pkg/reconciler/shared/tektonconfig/upgrade/upgrade.go

```diff
@@ -39,6 +39,7 @@ var (
 	// post upgrade functions
 	postUpgradeFunctions = []upgradeFunc{
 		upgradeStorageVersion, // upgrade #1: performs storage version migration
+		removeInstallerSets,   // Remove the installersets
```

Comment:
@pratap0007 nit: start a comment with `upgrade #2: removes the clusterTask installersets`